lab1/app.css

```diff
@@ -0,0 +1,62 @@
+button
+{
+    float: right;
+    color: black;
+    background-color: white;
+    font-size: 16px;
+    padding: 0.5em 1em;    
+}
+
+#story
+{ 
+  color: white; 
+  width:100%;
+  margin:5px;
+  padding:7px;
+} 
+#mainpara{
+    background-color: black;
+}
+}
+#resName {
+    font-size: 3em;
+    font-weight: bold;
+}
+#resleft{
+    float: left;
+}
+#container {
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  padding: 30px;
+}
+#a {
+    margin: 5px;
+    padding: 7px;
+    border: 2px solid black;
+    width: 33%;
+}
+#b {
+    margin: 5px;
+    padding: 7px;
+    border: 2px solid black;
+    width: 33%;
+}
+#c {
+    margin: 5px;
+    padding: 7px;
+    border: 2px solid black;
+    width: 33%;
+
+}
```

Comment:
These `ids` all looks the same, it's better off to refactor them into a single class so you can re-use the style between the elements. E.g. `.generator { ... }`

And in the HTML, you can use `class="generator"` rather than applying different ids.